src/PixelFlags.cc

```diff
@@ -94,22 +117,46 @@ PixelFlagsAlgorithm::PixelFlagsAlgorithm(Control const& ctrl, std::string const&
             schema.addField<afw::table::Flag>(name + "_flag_bad", "Bad pixel in the Source footprint");
     _anyKeys["SUSPECT"] = schema.addField<afw::table::Flag>(name + "_flag_suspect",
                                                             "Source's footprint includes suspect pixels");
-    // Flags that correspond to mask bits which occur in the center of the object
-    _centerKeys["INTRP"] = schema.addField<afw::table::Flag>(name + "_flag_interpolatedCenter",
-                                                             "Interpolated pixel in the Source center");
-    _centerKeys["SAT"] = schema.addField<afw::table::Flag>(name + "_flag_saturatedCenter",
-                                                           "Saturated pixel in the Source center");
-    _centerKeys["CR"] =
-            schema.addField<afw::table::Flag>(name + "_flag_crCenter", "Cosmic ray in the Source center");
-    _centerKeys["SUSPECT"] = schema.addField<afw::table::Flag>(name + "_flag_suspectCenter",
-                                                               "Source's center is close to suspect pixels");
+    // Flags that correspond to mask bits which are set anywhere in the 3x3 central region of the object.
+    _centerKeys["INTRP"] = schema.addField<afw::table::Flag>(
+            name + "_flag_interpolatedCenter", "Interpolated pixel in the 3x3 region around the centroid.");
+    _centerKeys["SAT"] = schema.addField<afw::table::Flag>(
+            name + "_flag_saturatedCenter", "Saturated pixel in the 3x3 region around the centroid.");
+    _centerKeys["CR"] = schema.addField<afw::table::Flag>(
+            name + "_flag_crCenter", "Cosmic ray in the 3x3 region around the centroid.");
+    _centerKeys["BAD"] = schema.addField<afw::table::Flag>(name + "_flag_badCenter",
+                                                           "Bad pixel in the 3x3 region around the centroid");
```

Comment:
I'm not sure why this is formatted differently than all of the other schema fields.

Reply:
Formatting is entirely determined by clang-format: in this case, I think it's because the docstring is shorter.